lib/v2/screens/sign_up/claim_invite/usecases/claim_invite_usecase.dart

```diff
@@ -28,4 +28,8 @@ class ClaimInviteUseCase {
 
     yield ClaimInviteMapper().mapInviteMnemonicToState(currentState, inviteMnemonic);
   }
+
```

Comment:
secretFromMnemonic() and hashFromSecret()
Import these methods from v2 instead v1 

`import 'package:seeds/v2/utils/mnemonic_code/mnemonic_code.dart';`